src/main/scala/codacy/plugins/test/ITest.scala

```diff
@@ -52,6 +52,7 @@ trait ITest {
       needsPatternsToRun = true, hasUIConfiguration = true) {
       override lazy val toolVersion: Option[String] = Some(dockerImageVersion)
       override val dockerTag: String = dockerImageVersion
+      override val dockerImageName: String = dockerImageFor(Option(dockerImageVersion))
```

Comment:
Eventually we should provide a way to override the tag from the constructor of the abstract class. For now this seems to be a valid hotfix